Dockerfile

```diff
@@ -1,25 +1,44 @@
-FROM centos:7
+FROM phusion/baseimage
```

Comment:
I thought we even had a scala baseimage too - might come with a jdk and sbt. I don't know where/if that exists, so this could be a red herring.